Doc/library/pdb.rst

```diff
@@ -217,6 +217,13 @@ access further features, you have to do this yourself:
    The *readrc* argument defaults to true and controls whether Pdb will load
    .pdbrc files from the filesystem.
 
+   The *mode* argument specifies how the debugger was invoked.
+   It impacts the workings of some debugger commands.
+   Valid values are ``'inline'`` (when called from the breakpoint() builtin),
+   ``'cli'`` (when called from a command line invocation)
```

Comment:
I'm not sure my last edit was quite right for the documentation, because users would call pdb how they want. The idea here is to indicate when the stdlib uses them. How about something like this?

```suggestion
   Valid values are ``'inline'`` (used by the breakpoint() builtin),
   ``'cli'`` (used by the command line invocation)
```